app/src/main/java/org/dharmaseed/android/TalkCursorAdapter.java

```diff
@@ -48,12 +58,29 @@ public void bindView(View view, Context context, Cursor cursor) {
 
         // Set talk title and teacher name
         TextView title=(TextView)view.findViewById(R.id.item_view_title);
-        TextView teacher=(TextView)view.findViewById(R.id.item_view_subtitle);
-        title.setText(cursor.getString(cursor.getColumnIndexOrThrow(DBManager.C.Talk.TITLE)).trim());
-        teacher.setText(cursor.getString(cursor.getColumnIndexOrThrow(DBManager.C.Teacher.NAME)).trim());
+        TextView teacher=(TextView)view.findViewById(R.id.item_view_detail1);
+        TextView center=(TextView)view.findViewById(R.id.item_view_detail2);
+        title.setText(cursor.getString(cursor.getColumnIndexOrThrow(DBManager.C.Talk.TABLE_NAME + "_" + DBManager.C.Talk.TITLE)).trim());
+        teacher.setText(cursor.getString(cursor.getColumnIndexOrThrow(DBManager.C.Teacher.TABLE_NAME + "_" + DBManager.C.Teacher.NAME)).trim());
+        center.setText(cursor.getString(cursor.getColumnIndexOrThrow(DBManager.C.Center.TABLE_NAME + "_" + DBManager.C.Teacher.NAME)).trim());
```

Comment:
I think that instead of manually doing the concat of `table`_`column` there should be a function for this in the respective DBManager.C class. Then we could just go something like `DBManager.C.Talk.getAlias(DBManager.C.Talk.Title)`. This way if we change what the alias is for whatever reason, it's centralized to that function and we don't need to make tons of changes.

Reply:
:+1: Agreed!  I'll add that.

Reply:
OK, added a `getAlias` method for generating the column aliases